analytics.mjs

```diff
@@ -58,7 +58,7 @@ const VIEW_IDS = {
   },
   v8: {
     viewId: '106306348',
-    pathRegexs: ['^/\\d{4}/\\d{2}/.*.html'],
+    pathRegexs: ['^/blog/'],
```

Comment:
Ideally, we’d get the combined Analytics for the old URL structure + the new one. (We still use the same tracking ID.) Is this possible without overcomplicating things? What I do manually is e.g. for `https://v8.dev/blog/10-years` I’d search analytics for the page slug `10-years`, which would also match the old URL `https://v8project.blogspot.com/2018/09/10-years.html`.